drivers/sensor/veml7700/veml7700.c

```diff
@@ -428,6 +447,9 @@ static int veml7700_sample_fetch(const struct device *dev,
 		} else {
 			return -ENOTSUP;
 		}
+	} else if ((enum sensor_channel_veml7700)chan == SENSOR_CHAN_VEML7700_WHITE_RAW_COUNTS) {
+		data = dev->data;
```

Comment:
Copy paste mistake? This like doesn't look necessary

Reply:
Good catch! Just fixed.